examples/plot/plot-layer/surface-layer.js

```diff
@@ -82,7 +82,8 @@ export default class SurfaceLayer extends Layer {
       vs: surfaceVertex,
       fs: fragmentShader,
       geometry: new Geometry({
-        drawMode: GL.TRIANGLES
+        drawMode: GL.TRIANGLES,
+        attributes: {}
```

Comment:
This seems to be a luma.gl bug (always issue a warning if `attributes` does not exit).

Reply:
Yeah, attributes can be set by attributeManger directly. We should check for possible attribute names in `opts` instead of `opts.attributes` to generate deprecated message.

Reply:
Supporting attributes outside of the `attributes` field became a little messy when subclassing `Geometry` so I added this warning. Was about to disable it, but this works too.